_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+[Pebtris](https://apps.rebble.io/en_US/application/52f9b7cd57e7c27b290000d7) | You'll have to hum your own theme music | No | Both
+
+..and some that have been released only recently:
+
+App | Description
+:--|:--
+[Hand wash timer](https://apps.rebble.io/en_US/application/5e9001813dd3102ef3c20a1a) | Make sure you wash your hands for long enough!
+[2019-Ncov](https://apps.rebble.io/en_US/application/5e65d05a3dd3108f341c08ec) | Track the number of Covid-19 cases around you
+[RSS Reader](https://apps.rebble.io/en_US/application/5d8d503bc393f5215a13e5e6) | Read the title and contents of your favourite RSS feeds
+[Hybrid Watchface](https://apps.rebble.io/en_US/application/5e463ef63dd31016ad6b17af) | A minimalistic watchace for Pebble Time and later
+[Sheikah Slate Watchface](https://apps.rebble.io/en_US/application/5e28ea933dd3109f151c7e0a) | The power of the sheikah slate on your wrist!
+
+<br>
+# Timeline
+June last year Rebble implemented timeline support. Which means you can once again push news, data, stocks, events and more to your timeline. It's a convenient way to get information you can about available quickly but unobtrusively. As some you already know, [I](https://willmurphy.co.uk/) wrote [a service](https://willow.systems/integrate-Pebble-with-ifttt-once-again/) that lets you hook anything from [If-This-Then-That](https://ifttt.com) to Pebble timeline, making it easy to push data.    
+
+As an example, I push the following data to my timeline:   
+
+Data  |  Method
+:--|:--
+BBC Breaking News | IFTTT: New Tweet to Timeline
+New post to /r/Pebble | IFTTT: Reddit to Timeline
+New RSS feed | IFTTT: RSS Feed applet to Timeline
+NASA Breaking News alerts | IFTTT: NASA to Timeline
+Monzo Debit card payments | [Timeline Bounce Server](https://github.com/Willow-Systems/timeline-bounce-server)
+New Hot item on hacker news | IFTTT: Hacker News to Timeline
+
+This keeps my timeline full of interesting stuff to read when I choosed to. Only BBC breaking news creates a notification, the rest is passive and I either read it when it pops up in [timeline peek](https://i.redd.it/7l87rqv4y9n41.png) or when I scroll back through timeline past (By mapping the 'up' quicklaunch to timeline past)   
+
+To get going with timeline you'll need [this app to generate a 'token'](https://apps.rebble.io/en_US/application/5d9ac26dc393f54d6b5f5445) and to follow [this guide](https://willow.systems/integrate-Pebble-with-ifttt-once-again/) or - if you prefer pictures - [this guide](https://imgur.com/a/IWG28JA). It helps if you're familiar with [If-This-Then-That](https://ifttt.com) too.
+
+# Developing New Apps
+Do you know how to program in C or Javascript? Maybe you want to learn? Developing on the Pebble is super fun and between the #app-dev channel on [the Discord](https://rebble.io/Discord) and [the documentation](https://developer.rebble.io/developer.Pebble.com/index.html) there's plenty of support still.   
+
```

Comment:
is it worth mentioning [the book](https://legacy.gitbook.com/book/pebble/learning-c-with-pebble/details) as another resource perhaps? :books: